### PR TITLE
Tidy up whitespace

### PR DESCRIPTION
**Pull request by asnare**
## Changes

This PR follows on from #1778 by cleaning up files that use tabs, have trailing whitespace, or are missing EOL at the end of files. By fixing these in one go other PRs become less noisy. (Sample files used in tests have been left alone and are not affected by this PR.)

### CODE REVIEW

**Review by JCZuurmond — CHANGES_REQUESTED** [review]
Is this the effect of changing a linter setting somewhere? The Changelog changes are a bit tricky, they will come. back

**Comment by asnare** [comment]
> Is this the effect of changing a linter setting somewhere?

It's a side-effect of the editor settings in #1778. Historically we have quite a few files with trailing whitespace and missing the EOL at EOF. These aren't enforced by linting (which only covers python code), but with the new editor settings in place any time you modify a file trailing whitespace elsewhere in the file will be stripped. This was intended as a fix-them-all-for-now PR to avoid spurious updates on other PRs, particularly in Markdown.

> The Changelog changes are a bit tricky, they will come back

Agreed; I'm happy to drop those given that the file is generated.

**Review by JCZuurmond — APPROVED** [review]

File: LICENSE [hunk 1]
```diff
@@ -1,69 +1,69 @@
                                Databricks License
                         Copyright (2023) Databricks, Inc.
 
-    Definitions. 
-    
-    Agreement: The agreement between Databricks, Inc., and you governing 
-    the use of the Databricks Services, as that term is defined in 
-    the Master Cloud Services Agreement (MCSA) located at 
+    Definitions.
```
Comment (nfx) [inline LICENSE:4]: please revert changes to this file

**Review by nfx — APPROVED** [review]
lgtm

**Pull request merged**